### PR TITLE
Documentation: Running ASPECT Overview

### PR DESCRIPTION
**Pull request by ljhwang**
This is only the first pass of fixing the converted text. More works still needs to be done as noted in the TODO as well as a second pass proofreading.

### CODE REVIEW

File: doc/sphinx/user/run-aspect/overview.md [hunk 1]
```diff
@@ -141,7 +140,8 @@ total 932
 -rw-rw-r-- 1 bangerth bangerth 577138 Dec 11 10:07 parameters.tex
 drwxr-xr-x 2 bangerth bangerth   4096 Dec 11 10:08 solution
 -rw-rw-r-- 1 bangerth bangerth    484 Dec 11 10:08 solution.pvd
--rw-rw-r-- 1 bangerth bangerth    451 Dec 11 10:08 solution.visit
+-rw-rw-r-- 1 bangerth bangerth    451 Dec 11 10:08 solution.
```
Comment (MFraters) [inline doc/sphinx/user/run-aspect/overview.md:143]: I don't think this change is correct.

File: doc/sphinx/user/run-aspect/overview.md [hunk 2]
```diff
@@ -88,24 +88,23 @@ Number of degrees of freedom: 33,859 (20,786+2,680+10,393)
...
+will discuss many such input files in {ref}`5`[]:
 
 ``` prmfile
 ```
 
 In other words, these run-time parameters specify that we should start with a
 geometry that represents a spherical shell (see
-Sections&nbsp;{ref}`parameters:Geometry_20model`48] and
-{ref}`parameters:Geometry_20model/Spherical_20shell`49] for details). The
+Sections&nbsp;{ref}`parameters:Geometry_20model`] and
```
Comment (MFraters) [inline doc/sphinx/user/run-aspect/overview.md:106]:
```suggestion
Sections {ref}`parameters:Geometry_20model` and
```

File: doc/sphinx/user/run-aspect/overview.md [hunk 3]
```diff
@@ -14,15 +14,15 @@ or, if you want to run the program in parallel, using something like
       mpirun -np 4 ./aspect parameter-file.prm
 
 to run with 4 processors. In either case, the argument denotes the (path and)
-name of a file that contains input parameters.[7] When you download
+name of a file that contains input parameters.[^footnote1] When you download
 ASPECT, there are a number of sample input files in
 the `cookbooks` directory, corresponding to the examples discussed in
-{ref}`5][], and input files for some of the benchmarks discussed in
+{ref}`5`][], and input files for some of the benchmarks discussed in
```
Comment (MFraters) [inline doc/sphinx/user/run-aspect/overview.md:20]:
```suggestion
{ref}`5`[], and input files for some of the benchmarks discussed in
```

**Review by MFraters — CHANGES_REQUESTED** [review]
I think you accidentally deleted something (the visit extension), could you restore that?

File: doc/sphinx/user/run-aspect/overview.md (hunk 1 above)
Comment (bangerth) [reply doc/sphinx/user/run-aspect/overview.md:143]: I stared at this for a good minute trying to figure out what I had done wrong, seeing my name all over that text but not recalling having anything to do with this PR. Then I realized that I had nothing to do with what is discussed here except that I'm being used as an example :-)

Comment (gassmoeller) [inline doc/sphinx/user/run-aspect/overview.md:143]:
```suggestion
-rw-rw-r-- 1 bangerth bangerth    451 Dec 11 10:08 solution.visit
```

**Review by gassmoeller — APPROVED** [review]
I applied the suggestions and resolved the merge conflict. Should be ready to go. @MFraters are you ok with the current state? If so, please approve.

**Review by MFraters — APPROVED** [review]
looks good to me!

File: doc/sphinx/user/run-aspect/overview.md [hunk 4]
```diff
@@ -284,13 +284,51 @@ therefore also in MKS units.
     visualized, see for example Figure&nbsp;[2][51]. The plot shows how an
     initially linear temperature profile forms upper and lower boundary
     layers.
+
```
Comment (gassmoeller) [inline doc/sphinx/user/run-aspect/overview.md:287]:
```suggestion

```

**Comment by bangerth** [comment]
This fails the indentation tester. Can you fix that, @ljhwang ?

**Comment by ljhwang** [comment]
I will close this branch as well and start a new one due to the unresolved conflicts.

**Pull request closed without merging**